### ISSUE TITLE
[EDGE][BROWSE MODE] NVDA cant read plain text on a div

### ISSUE THREAD

**Issue by MarxAblir** [nvaccess/nvda#5914]
Upon hover on text NVDA does not read the text on div containers unlike in other browsers

**Comment by michaelDCurran** [comment]
@MarxAblir: Please provide an example file and steps to reproduce.

**Comment by Adriani90** [comment]
@MarxAblir we cannot reproduce this issue without exact steps to reproduce and an example website or an example file. Could you please provide these details? Thanks.

**Comment by Adriani90** [comment]
Closing as abandoned and works for me. I cannot reproduce this in Edge Chromium 81.
